.vscode/launch.json

```diff
@@ -2,40 +2,13 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "name": "Cortex-Debug J-Link",
-      "preLaunchTask": "Build Debug",
-      "type": "cortex-debug",
-      "request": "launch",
-      "cwd": "${workspaceRoot}",
-      "executable": "${workspaceRoot}/build/Debug/deluge.elf",
-      "servertype": "jlink",
-      "device": "R7S721020",
-      "interface": "swd",
-      "runToEntryPoint": "main",
-      "svdFile": "${workspaceRoot}/contrib/rza1.svd",
-      "rttConfig": {
-        "enabled": true,
-      },
-    },
-    {
-      "name": "Cortex-Debug OpenOCD",
-      "preLaunchTask": "Build Debug",
-      "type": "cortex-debug",
+      "name": "Debug Tests",
+      "type": "cppdbg",
       "request": "launch",
+      "program": "${workspaceRoot}/tests/RunAllTests",
+      "MIMode": "gdb",
       "cwd": "${workspaceRoot}",
-      "executable": "${workspaceRoot}/build/Debug/deluge.elf",
-      "serverpath": "${workspaceRoot}/toolchain/win32-x64/openocd/bin/openocd.exe",
-      "servertype": "openocd",
-      "configFiles": [
-        "/interface/cmsis-dap.cfg",
-        "${workspaceRoot}/contrib/renesas-rz-a1lu.cfg",
-      ],
-      "device": "R7S721020",
-      "runToEntryPoint": "main",
-      "svdFile": "${workspaceRoot}/contrib/rza1.svd",
-      "rttConfig": {
-        "enabled": true,
-      },
+      "miDebuggerPath": "gdb",
```

Comment:
Did you mean to fully replace this launch config with the run tests one?

Reply:
Oops, deleted them locally forever ago and forgot that would get checked in. I'll restore them